### ISSUE TITLE
wp-cli.yml with ssh not using `path`

### ISSUE THREAD

**Issue by zanona** [wp-cli/wp-cli#5324]
Strangely enough I can get this to work without problem:

```bash
wp --ssh=ssh-config-alias --path=path/to/wp-on-remote-server rewrite flush
```

but if I create a `wp-cli.yml` on my project root with the following:
```yml
ssh: ssh-config-alias
path: path/to/wp-on-remote-server
```
I then get:
```bash
$ wp rewrite flush
Error: This does not seem to be a WordPress installation.
Pass --path=`path/to/wordpress` or run `wp core download`.
```
It seems that the config file is only applying the `ssh` option but not `path` as If I then run the below, it works.

```sh
wp --path=path/to/wp-on-remote-server rewrite flush 
```
So apparently, I am unable to get `wp-cli.yml` working with SSH.

**Comment by aaemnnosttv** [comment]
> `wp --ssh=ssh-config-alias --path=path/to/wp-on-remote-server`

This doesn't work for me, but looks like it probably should.

For SSH commands I find you have to include the path as part of the SSH connection string. See the docs for this here: https://make.wordpress.org/cli/handbook/running-commands-remotely/#using-an-ssh-connection.

Try updating your config like so:
```yml
ssh: ssh-config-alias/absolute/path/to/wp-on-remote-server
```

The difference in resulting SSH commands are as follows, as observed when adding `--debug`

Running `wp option get home --ssh=ssh-alias --path=/path/to/wp/` (doesn't work for me)
```
Running SSH command: ssh -q -t 'ssh-alias' 'wp '\''option'\'' '\''get'\'' '\''home'\'' '\''--path=/path/to/wp/'\'' '\''--debug'\''' 
```

Running `wp option get home --ssh=ssh-alias/path/to/wp/` (works)
```
Running SSH command: ssh -q -t 'ssh-alias' 'cd '\''/path/to/wp/'\''; wp '\''option'\'' '\''get'\'' '\''home'\'' '\''--debug'\'''
```

Honestly, I would expect both to work so there seems to be at least room for improvement here (at least in terms of documentation).

**Comment by zanona** [comment]
Hey, @aaemnnosttv thanks for the suggestion. I will try it out. However, I have been able to get it working by simply labelling the environment like so:

### wp-cli.yml
```yml
@dev:
  ssh: ssh-config-alias-dev
  path: path/to/wp-on-remote-server
@live:
  ssh: ssh-config-alias-live
  path: path/to/wp-on-remote-server
``` 

With that, I can now use `wp @dev rewrite flush` without a problem. It turns out, this feels actually better and more organised, what do you reckon? :)

I will leave up to you guys closing or maintaining this issue in case you see room for improvement as you said.

Thanks again.

**Comment by danielbachhuber** [comment]
> Honestly, I would expect both to work so there seems to be at least room for improvement here (at least in terms of documentation).

I don't think we should change the existing behavior, but additional documentation seems fine.
